node_modules/oae-core/sharedwith/js/sharedwith.js

```diff
@@ -26,21 +26,22 @@ define(['jquery', 'oae.core'], function($, oae) {
         /**
          * Initialize a new infinite scroll container that fetches the members
          *
-         * @param  {Discussion}    discussionProfile    Discussion for which the members should be rendered
+         * @param  {Content|Discussion|Folder}    contextProfile    Entity for which the members should be rendered
```

Comment:
We don't really have a reliable process to come back and update this for each resource type that gets added in the future. We should probably just specify `Object` and indicate it accepts any resource model with a `resourceType`, `id` and a members list